src/chi/components/list-group/list-group.scss

```diff
@@ -0,0 +1,32 @@
+@import 'variables';
+@import 'mixins';
+
+$border: 0.0625rem;
+
+.m-listGroup {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+
+  & li {
+    border: $border solid set-color(grey, 30);
```

Comment:
@davidojedacl can you add this property and remove this to the .m-listGroup class